src/NUnitConsole/nunit3-console.tests/nunit3-console.tests.csproj

```diff
@@ -13,7 +13,6 @@
   <ItemGroup>
     <PackageReference Include="NSubstitute" Version="2.0.3" />
     <PackageReference Include="NUnit" Version="3.11.0" />
-    <PackageReference Include="NUnit.Analyzers" Version="0.1.0-dev-00079" />
```

Comment:
Good illustration here. Dropping the analyzers got checked in by mistake. I generally drop them since I can't build in my admittedly strange environment, but avoid committing that change. It's another step to remember to take.

We dogfooded a lot at Microsoft, but it always meant using __production__ tools. For NUnit, we test the framework using the framework under development but using a released console or adapter. We test the adapter using a released framework, etc. That way we are testing only one thing at a time. The analyzer is the only exception to that I know of. OTOH, I'd really like to know why it causes a versioning conflict in my configuration that @mikkelbu can't duplicate.